src/components/auth-components/login-form.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Field, reduxForm, focus } from 'redux-form';
+import Input from './input';
+import { login } from '../../actions/auth';
+import { required, nonEmpty } from '../../validators';
+import './login-form.css';
+export class LoginForm extends React.Component {
+  onSubmit(values) {
+    return this.props.dispatch(login(values.username, values.password));
+  }
+
+  render() {
+    // console.log(this.props);
+    let error;
+    if (this.props.error) {
+      //   console.log('error:',this.props.error);
+      error = (
+        <div className="form-error" aria-live="polite">
+          {this.props.error}
+        </div>
+      );
+    }
+    return (
+      <form
+        className="login-form"
+        onSubmit={this.props.handleSubmit(values => this.onSubmit(values))}
+      >
+        {error}
+        <label htmlFor="username">Username</label>
+        <Field
+          component={Input}
+          // component='input'
+          type="text"
+          name="username"
+          id="username"
```

Comment:
I don't think you need to pass the `id` prop, since the `Input` will set this for you (equal to `name`)